solution/ui/e2e/cypress/e2e/policy-repository.spec.cy.js

```diff
@@ -42,10 +42,10 @@ Cypress.Commands.add("getPolicyDocs", ({ username, password }) => {
 describe("Policy Repository", () => {
     beforeEach(_beforeEach);
 
-    it("shows the login screen when you visit /policy-repository/ without logging in", () => {
+    it("shows the custom eua login screen when you visit /policy-repository/ without logging in", () => {
```

Comment:
if we're going to include all of the header functionality on the custom login page, perhaps we should test that the stuff works. 

You could copy and reuse the Error Page test suite (`error-page.cy.js`) and remove all the error page stuff and replace it with visiting the login page.  It should check for a11y compliance as well as making sure the flash banner, feedback modal and Jump To functionality works as expected.